src/cfnlint/custom_rules.py

```diff
@@ -3,17 +3,18 @@
 SPDX-License-Identifier: MIT-0
 """
 # pylint: disable=W0108
-# pylint: disable=W0622
+
 import logging
 import cfnlint
+import cfnlint.runner
 import cfnlint.customRules.Operators
 import cfnlint.customRules.Rule
 
 LOGGER = logging.getLogger(__name__)
-Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y),
+Operator = {'EQUALS': lambda x, y, z: cfnlint.customRules.Operators.equalsOp(x, y, z),
             'PLACEHOLDER': lambda x, y: LOGGER.debug('Placeholder Op')}
 
-def check(filename, template):
+def check(filename, template, rules, runner):
```

Comment:
still called with only the two arguments in the unit test from last PR:

https://github.com/aws-cloudformation/cfn-python-lint/commit/7d83cf60d4348056ae71b4ca8d46f06fbb2d281d#diff-720eee6a33e1ef12bafaf682cb58ccb2R66

https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/346855763#L363

Reply:
Corrected